models/examples/netbox/netbox.yml

```diff
@@ -685,7 +685,7 @@ nodes:
     display_labels:
       - prefix__value
     uniqueness_constraints:
-      - ["vrf", "prefix__value"]
+      - ["prefix__value"]
```

Comment:
So this would basically mean we can't have overlapping prefixes anymore, even if they are assigned to different namespace?

Reply:
in this example we are not using the BuiltinIPPrefix so we are not using namespace (non existent in Netbox)